app/models/comment.rb

```diff
@@ -1,3 +1,7 @@
 class Comment < ActiveRecord::Base
   belongs_to :commentable, :polymorphic => true
+
+  scope :recent, ->{ order('created_at DESC') }
+
+  validates_presence_of :person_id, :description
```

Comment:
You might not have noticed, but this is a polymorphic class. I don't think we can validate on person_id.

Reply:
I believe `person_id` will validate (we should always have the concept of who posted the comment?) since that's stored in the comments table itself. At least I can't think of any reason why it wouldn't, but also haven't touched a polymorphic association in a while. 

I did think that `person_id` might better be called `poster_id`. That might be confusing for when the commentable type and id are a Person though, so... idk. I'll remove the validation if it ends up being a 💣 while implementing create comments.